### PR TITLE
#184948246 Remove semaphore and mantainability budge

### PR DESCRIPTION
**Pull request by Rwibutso**
What does this PR do?
This PR remove usemaphore and mantainability budge on our repo
Description of Task to be completed?
• Rmove semaphore and mantainability budge
How should this be manually tested?
• clone this repo git clone https://github.com/atlp-rwanda/e-comm-team-amigos-fn
• switch your working directory cd e-comm-team-amigos-fn
• install all dependencies by running npm install
• make any relevant change and pushAny background context you want to provide?
• Have have up-to-date browser
• Have Javascript enabled in browser
What are the relevant pivotal tracker/Trello stories?
Story ID: [#184948246](https://www.pivotaltracker.com/story/show/184948246)

### CODE REVIEW

File: dist/.github/workflows/test.yml
Comment (khevin2) [inline dist/.github/workflows/test.yml:1]: Put this file in `./github/workflows`

**Review by nshutijonathan — APPROVED** [review]
LGTM

**Review by AbdulKhaliq59 — APPROVED** [review]
LGTM

**Review by Habinezajanvier — APPROVED** [review]
👍

**Pull request merged**